### ISSUE TITLE
You need to have both enabled WP_Debug and Gantry 5 debug for dump to work

### ISSUE THREAD

**Issue by newkind** [gantry/gantry5#839]
Setting Gantry 5 in debug mode should be enough to get dump() working.

**Comment by mahagr** [comment]
The issue is that we are calling Timber::render() from our templates and not directly our rendering functions.

Moving into future milestone.

**Comment by w00fz** [comment]
I think this is now fixed with the new DebugBar?

**Comment by mahagr** [comment]
Not really, we still need to add support for `dump()` to render in debugbar. I believe Grav already implemented something like this?
